doc/source/ray-use-cases/using-ray-with-ros.rst

```diff
@@ -0,0 +1,51 @@
+Using Ray with the Robotics Operating System
+============================================
+
+The `Robotics Operating System (ROS) <https://www.ros.org/>`__ is a set of software libraries and tools
+that help you build robot applications. You can use Ray to run your ROS application in a distributed way,
+for example by:
+
+- parallelizing simulations,
+- data processing for preparing training data using ROS libraries,
+- running reinforcement learning with a ROS environment
+
+Setting up the environment
+--------------------------
+
+The ROS distribution consists of a collection of libraries and we need to make sure they can be found
+by Ray's worker processes. ROS typically uses a bash script `/path/to/ros/setup.bash` to
+set up the required environment variables and you can propagate this information to Ray using
+:ref:`runtime environments <runtime-environments>` like this
+
+.. code-block:: python
+
+    import subprocess
+    import ray
+
+    # Set up ROS environment for the Ray driver process
+    output = subprocess.check_output("source /path/to/ros/setup.bash; env -0", shell=True, executable="/bin/bash")
```

Comment:
Could we have a) super quick installation instructions for ROS etc., and b) a very simple, running script? Would be great to start out with something executable for this new section (and test the snippet in CI).

Reply:
Thanks, I actually removed the example now. The use case that motivated this was actually solved differently in the end, I think there is less value in having this. Will keep it in mind for the future if it comes up again :)